README.md

```diff
@@ -1 +1,142 @@
 # servo-rancher
+Optune servo driver for Rancher (v1.6)
+
+This driver supports an application deployed via Rancher as a Stack.
+
+A Rancher Stack may contain multiple containers, each of which may individually be modified by the adjust driver.
+Each container in the Stack is considered a separately adjustable component.
+
+For each component of the application, the following settings are automatically available when 'adjust --info' is run:
+* scale (number of container instances)
+* memory reservation
+* milli-cpu reservation
+* environment variables
+
+A configuration file must be present to define scale of variable modification config.yaml, any custom component settings in it are also returned by 'adjust --info' and map to environment variables of the matching container.
+
+Currently the following parameters are supported:
```

Comment:
Does the proposed version support arbitrary ENV vars or just a predefined list? Either way, I would list the fixed variables first, i.e. (cpu, mem, scale) and then a section with environment vars -  either the list of the supported vars or if this is a generic method, explain how env vars are distinguished from fixed vars (i.e. If I have a setting called mem is that going to be treated as a reservation or an env var, what is the logic/limitations behind this).

Reply:
we don't limit them.